Pipfile

```diff
@@ -66,6 +66,7 @@ braintree = "==3.49.0"
 razorpay = ">=1.1.1"
 stripe = "*"
 "urllib3" = "*"
+dataclasses = "*"
```

Comment:
Kindly remind that requirements files should also be updated by ```pipenv lock -r > requirements.txt```

Reply:
Travis kindly reminded him by failing 😉